contributing/deprecation_policy.md

```diff
@@ -38,51 +128,70 @@ A three-month deprecation is similar with more time for clients to adjust code:
 
 Map this schedule onto reality: 
 
-1. Is there a holiday/conference coming up that would prevent clients from being able to collaborate and comment? 
+1. Is there a holiday/conference coming up that would prevent clients from being able to
+   collaborate and comment? 
 1. Is the to-be-replaced code particularly problematic for some reason? 
 1. Does the deprecation fall near a release of the operating system or new devices?
 
```

Comment:
Should we try to group "breaking changes" of different APIs together by designating preferred deletion dates and possibly preferred versions for breaking changes? It might make it easier for our client base to plan ahead for these dates/versions.  We might also want avoid mixing breaking changes with major new functionality in a single release, so it's easier for client apps to rollback breaking changes.

Reply:
Good topics to consider.

If we follow this process fully, breaking changes will be mitigated for client teams because by the time we ship our major release with the APIs deleted, all of our internal clients will have already migrated off of the APIs and impact will be minimal.

That being said, this is less the case for our external clients. I do think it's fair to consider lumping multiple deletions together into a single release. How do you feel we might we coordinate API deprecations in this manner?